kubernetes-manifests/cartservice.yaml

```diff
@@ -27,8 +27,20 @@ spec:
     spec:
       serviceAccountName: default
       terminationGracePeriodSeconds: 5
+      securityContext:
+        fsGroup: 1000
+        runAsGroup: 1000
+        runAsNonRoot: true
```

Comment:
Question: Is `runAsNonRoot` adding anything here (possibly redundant) since we already specify `runAsUser: 1000`?
I assume `runAsUser: 0` would be the `root` User (since [`runAsGroup: 0` is the root Group](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#:~:text=If%20this%20field%20is%20omitted%2C%20the%20primary%20group%20ID%20of%20the%20containers%20will%20be%20root(0).)).

Reply:
That's a good point, never thought about that. Yep agree, seems redundant, but at least more explicit and not hurting anyway? Let's keep this like this and see in the future if we want to change that. Thanks for bringing this up! Good point.